src/main/java/hudson/plugins/cppncss/CppNCSSPublisher.java

```diff
@@ -102,5 +135,40 @@ public boolean isApplicable(Class<? extends AbstractProject> aClass) {
             return CppNCSSHealthMetrics.values();
         }
     }
+    
+    /** This isn't actually an action itself, but only a way to get 
+     * project actions through {@link ActionGetter#getProjectActions}. Therefore all the 
+     * methods implemented from {@link Action} return null.
+     *
+     */
+    protected static class ActionGetter implements SimpleBuildStep.LastBuildAction {

```

Comment:
Why do you need it?

Reply:
Could not find a way to get it to work properly without this.